README.md

```diff
@@ -587,9 +587,6 @@ An HTTP(S) response. This class implements the [Body](#iface-body) interface.
 
 The following properties are not implemented in node-fetch at this moment:
 
-- `Response.error()`
```

Comment:
@tkrotoff Please add a note in an appropriate place in the README that only `default` and `error` response types are used and that node-fetch does not make use of filtered responses https://fetch.spec.whatwg.org/#concept-filtered-response

Reply:
Done.

```
#### response.type

<small>_(deviation from spec)_</small>

Convenience property representing the response's type. node-fetch only supports `'default'` and `'error'` and does not make use of [filtered responses](https://fetch.spec.whatwg.org/#concept-filtered-response).
```